FIAT/argyris.py

```diff
@@ -53,13 +53,17 @@ def __init__(self, ref_el, degree, variant, interpolant_deg):
             # interior dofs
             q = degree - 6
             if q >= 0:
-                Q = create_quadrature(ref_el, interpolant_deg + q)
-                Pq = polynomial_set.ONPolynomialSet(ref_el, q, scale=1)
-                phis = Pq.tabulate(Q.get_points())[(0,) * sd]
-                scale = ref_el.volume()
-                cur = len(nodes)
-                nodes.extend(IntegralMoment(ref_el, Q, phi/scale) for phi in phis)
-                entity_ids[sd][0] = list(range(cur, len(nodes)))
+                cell = ref_el.construct_subelement(sd)
```

Comment:
Does macro Argyris make sense if we're not zany mapping the reference Argyris to each subcell?

Reply:
Hence, having a FIAT-only zany facility would be very helpful for testing as well as tiling.  Bu tI agree, it can be a separate PR.  Maybe a comment about limitations and/or just raising exceptions if we attempt to tile known-to-fail elements?